Makefile

```diff
@@ -7,6 +7,8 @@ lint:
 	pip install .[lint]
 	isort .
 	black .
+	pyreverse pymc_marketing/mmm -d docs/source/uml -f 'ALL' -o png -p mmm
```

Comment:
Shall we add `pyreverse` to the list of dev-packages or is it from `pylint` below ?

Reply:
It's from `pylint`

Reply:
thanks!